gprofiler/main.py

```diff
@@ -575,6 +589,12 @@ def verify_preconditions(args):
         print("Could not acquire gProfiler's lock. Is it already running?", file=sys.stderr)
         sys.exit(1)
 
+    if args.log_cpu_usage and get_run_mode() not in ("k8s", "container"):
+        # TODO: we *can* move into another cpuacct cgroup, to let this work also when run as a standalone
+        # executable.
+        print("--log-cpu-usage is available only when run as a container!")
```

Comment:
Hmm, maybe exit and fail the execution of the gProfiler?

Reply:
Next line is `sys.exit(1)` lol